### PR TITLE
Newcandidate 371rc6

### PR DESCRIPTION
**Pull request by mdlpstsci**
Resolves [HLA-nnnn](https://jira.stsci.edu/browse/HLA-nnnn)

Closes #

Cherry picked changes from Drizzlepac main onto the release branch 3.7.1x.

**Checklist for maintainers**
- [ ] added entry in `CHANGELOG.rst` within the relevant release section
- [ ] updated or added relevant tests
- [ ] updated relevant documentation
- [ ] added relevant label(s)
- [ ] ran regression tests, post a link to the Jenkins job below.
      [How to run regression tests on a PR](https://github.com/spacetelescope/jwst/wiki/Running-Regression-Tests-Against-PR-Branches)

### CODE REVIEW

File: drizzlepac/haputils/deconvolve_utils.py [hunk 1]
```diff
@@ -479,7 +479,7 @@ def _create_input_psf(psf_name, calimg, total_flux):
     lib_size = [lib_psf_arr.shape[0] // 2, lib_psf_arr.shape[1] // 2]
 
     # create hamming 2d filter to avoid edge effects
-    h = ss.hamming(lib_psf_arr.shape[0])
+    h = windows.hamming(lib_psf_arr.shape[0])
```
Comment (mcara) [inline drizzlepac/haputils/deconvolve_utils.py:482]: is this backward compatible?

Comment (mdlpstsci) [reply drizzlepac/haputils/deconvolve_utils.py:482]: Yes.

Comment (mdlpstsci) [reply drizzlepac/haputils/deconvolve_utils.py:482]: @mcara It is just an import and namespace change.  Please see PR#1848.

Comment (mcara) [reply drizzlepac/haputils/deconvolve_utils.py:482]: Why not use the similar function from `numpy`?

Comment (mcara) [reply drizzlepac/haputils/deconvolve_utils.py:482]: https://numpy.org/doc/stable/reference/generated/numpy.hamming.html

Comment (mdlpstsci) [reply drizzlepac/haputils/deconvolve_utils.py:482]: @mcara I could consider this for the future!  However, we are in the middle of a build sequence, so I am not inclined to change anything too dramatically at the last minute.

**Review by mcara — APPROVED** [review]

**Pull request closed without merging**